deephaven_ipywidgets/_version.py

```diff
@@ -4,5 +4,5 @@
 # Copyright (c) Mike Bender.
 # Distributed under the terms of the Modified BSD License.
 
-version_info = (0, 1, 0, 'dev')
+version_info = (0, 1, 1)
```

Comment:
We are we bumping from 0.1.0.dev0 to 0.1.1? Should we go to `0.1.0.dev1`, `0.1.0`, or `0.1.1.dev0` instead?

Reply:
Yea I guess 0.1.0 makes more sense. Also - I should do the rename the JS package from `deephaven-ipywidgets` to `@deephaven/ipywidgets` now as well, so that the packages show up under the deephaven scope correctly: https://www.npmjs.com/org/deephaven

Then I can grant deephaven-bot permissions to publish to the deephaven org.